inc/Engine/Media/Lazyload/Content/Processor/Dom.php

```diff
@@ -0,0 +1,68 @@
+<?php
+declare(strict_types=1);
+
+namespace WP_Rocket\Engine\Media\Lazyload\Content\Processor;
```

Comment:
@Tabrisrp Actually, this feature is about lazyrender, and not lazyload. It could be good to change the namespace and folder name for this feature before we actually start more work on it.